rfc

```diff
@@ -24,11 +24,11 @@ __rfc() {
     local NO_CURL_WGET=4
 
     # URLs
-    local RFCS_TGZ_BASE_FTP='ftp://ftp.rfc-editor.org/in-notes/tar/'
+    local RFCS_TGZ_BASE_FTP='https://ftp.rfc-editor.org/in-notes/tar/'
```

Comment:
Mind renaming this to `RFCS_TGZ_BASEURL` while you’re here? It doesn’t make sense to have something named `…_FTP` if that’s not `ftp` anymore.